projects/framework-common-extensions/plugins/resolve_entity_path.py

```diff
@@ -0,0 +1,81 @@
+# :coding: utf-8
+# :copyright: Copyright (c) 2024 ftrack
+import sys
+
+from ftrack_utils.string import str_version
+from ftrack_framework_core.plugin import BasePlugin
+from ftrack_framework_core.exceptions.plugin import PluginExecutionError
+import clique
+
+
+class ResolveEntityPathsPlugin(BasePlugin):
+    name = 'resolve_entity_path'
+
+    def _resolve_entity_paths(self, options):
+        '''Evaluate list of entities passed on in 'options', ensure
+        a single component and resolve path. Return as dictionary'''
+        result = {}
+        entities = options.get('selection', [])
+        if not entities:
+            raise PluginExecutionError('No entities selected!')
```

Comment:
Consider removing the exlamation mark (!) and just use a dot: "No entities selected."
Git gives a bit more of a formal tone in the error message.

Reply:
Yes, sorry I didn't saw it when reviewing the code. Changed now.